WalletWasabi.Fluent/ViewModels/ApplicationViewModel.cs

```diff
@@ -38,6 +39,22 @@ public ApplicationViewModel()
 
 		ShowCommand = ReactiveCommand.Create(() => ShowRequested?.Invoke(this, EventArgs.Empty));
 
+		AboutCommand = ReactiveCommand.Create(  () =>
+		{
+			var dialogScreen = MainViewModel.Instance?.DialogScreen;
+
+			if (dialogScreen is null)
+			{
+				return;
+			}
```

Comment:
`MainViewModel.Instance` is not nullable. remove the `?` mark and the null check.